package.json

```diff
@@ -25,6 +25,7 @@
     "axios": "^0.21.1",
     "bootstrap": "^4.4.1",
     "bootswatch": "^4.5.0",
+    "classnames": "^2.2.6",
```

Comment:
`classnames` is added here, but seemingly no corresponding `package-lock.json` update in this PR? Is that correct?